src/decaylanguage/dec/enums.py

```diff
@@ -95,4 +96,5 @@ class PhotosEnum(IntEnum):
     "VVP",
     "VVS_PWAVE",
     "YMSTOYNSPIPICLEO",
+    "YMSTOYNSPIPICLEOBOOST",
```

Comment:
```suggestion
```

Reply:
Unless it changed the longer string needs to go first otherwise the parser only finds the substring. This is easily checked with a mini test file using the longer name as model, and see if it is found.

Reply:
I don't think this is necessary anymore. The grammar requires now that the model name is followed by at least one white space character or a semicolon (I think that's more or less what evtgen does but don't quote me on that) so it will match the correct model name regardless. However I will run some tests to ensure that what I'm saying is true.

Reply:
Cool if that's the case now. That will be an improvement in spite of the stricter requiremen of a WS or SC. Better check but if OK then simply reject my changes. Thanks.

Reply:
This works, I've added a test to ensure this too.

Reply:
Many thanks.